subt/ros/proxy/ros_proxy_node.cc

```diff
@@ -376,8 +379,7 @@ not available.");
              pose.orientation.x, pose.orientation.y, pose.orientation.z, pose.orientation.w);
 
   // Distance to goal
-  double dist = pose.position.x * pose.position.x +
-    pose.position.y * pose.position.y;
+  double dist = sqrt(pose.position.x * pose.position.x + pose.position.y * pose.position.y);
 
   if(abs(prev_dist2 - dist) > 1.0)
```

Comment:
here the `prev_dist2` was meant as "sqare"

Reply:
`prev_dist2` is `dist` one step back and comparing distance or distance squared with 1 should lead to the the same result